packages/datadog-plugin-cucumber/src/index.js

```diff
@@ -68,7 +68,7 @@ class CucumberPlugin extends CiPlugin {
         }
       )
       if (isEarlyFlakeDetectionEnabled) {
-        this.testSessionSpan.setTag(TEST_EARLY_FLAKE_IS_ENABLED, 'true')
+        this.testSessionSpan.setTag(TEST_EARLY_FLAKE_ENABLED, 'true')
```

Comment:
is it by design that threshold logic is only added to jest? or do other test frameworks work via jest runner?

Reply:
yes, this PRs only adds the logic to jest. I'll update the PR description as it's confusing